### ISSUE TITLE
In transactions.cpp parseParams needs to be removed

### ISSUE THREAD

**Issue by tjayrush** [Great-Hill-Corporation/quickBlocks#42]
From https://github.com/Great-Hill-Corporation/ethslurp/issues/130

This code was a very early way to parse functions but it is totally wrong. It has all been subsumed by the monitors, but does have the benefit of not requiring a separate piece of software. In other words remove it, but don't lose the ability to parse a totally arbitrary input data the way ethslurp does. But also don't require a full-fledged monitor program.

In other words--input parsing should happen fully through the ABI if that is the only thing present, but through the parse library if that is what is needed for speed or flexibility. Note that monitors use this code to fill in the [{FUNCTION}] tag.

**Comment by tjayrush** [comment]
Referencing #51

**Comment by tjayrush** [comment]
Closing for now. Will reopen once 0.2.0 is finished. Referenced by #293.
